src/tracing/group_keys.h

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
It would be good to evaluate if unused strings are optimized out at compile time with the array